lib/python/rose/apps/comparisons/within.py

```diff
@@ -23,6 +23,8 @@
 from rose.apps.rose_ana import DataLengthError
 
 OUTPUT_STRING = "%s %s%% %s %s: File %s c.f. %s (%s values)"
+OUTPUT_STRING_WITH_POSN = "%s %s%% %s %s: File %s c.f. %s (value %s of %s)"
+OUTPUT_STRING_WITH_VALS = "%s %s%% %s %s: File %s (%s) c.f. %s (%s)"
```

Comment:
Perhaps it is worth using named arguments % dict instead of positional arguments % tuple?